dev/com.ibm.ws.jdbc/src/com/ibm/ws/jdbc/internal/JDBCDriverService.java

```diff
@@ -331,7 +331,7 @@ public T run() throws Exception {
      * <li>javax.sql.ConnectionPoolDataSource
      * <li>javax.sql.DataSource
      * <li>javax.sql.XADataSource
-     * <li>java.sql.Driver // TODO
+     * <li>java.sql.Driver
```

Comment:
It is incorrect to place Driver in this ordered list of data source precedence due to our reliance on the URL attribute

Reply:
I was just removing the TODO but I can remove the entry from the list entirely